### PR TITLE
Use matplotlib coverage plot/genome diagram

### PR DESCRIPTION
**Pull request by christinehe**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -55,6 +51,23 @@ RUN git clone https://github.com/cov-lineages/pangolin.git \
...
+
+RUN pip install nbconvert==6.0.7
+
+# Install onecodex_pdf export option
+RUN pip install onecodex[all,reports]==v0.9.4
+RUN mkdir -p /usr/local/share/fonts \
+    && cp /usr/local/lib/python3.8/site-packages/onecodex/assets/fonts/*.otf /usr/local/share/fonts \
+    && fc-cache
+
+RUN npm install md5
```
Comment (audy) [inline Dockerfile:69]: Is this needed?

File: jobscript.sh [hunk 2]
```diff
@@ -55,8 +55,10 @@ cp /share/low_complexity_regions.txt .
 echo "Generating notebook!"
 
 RESULTS_DIR="$(pwd)" \
+
```
Comment (audy) [inline jobscript.sh:58]: This blank line breaks the jobscript

**Review by audy — CHANGES_REQUESTED** [review]
Two changes requested

File: Dockerfile [hunk 3]
```diff
@@ -7,14 +7,10 @@ RUN pip install numpy
...
 
-RUN pip install onecodex[all,reports]==v0.9.4
-
-RUN mkdir -p /usr/local/share/fonts \
-    && cp /usr/local/lib/python3.8/site-packages/onecodex/assets/fonts/*.otf /usr/local/share/fonts \
-    && fc-cache
 
 USER root
 RUN apt-get update \
+    && apt-get autoclean \
```
Comment (audy) [inline Dockerfile:13]: is this needed?

**Comment by christinehe** [comment]
BIO-277

**Pull request merged**